### PR TITLE
Adds tools of quality

### PR DESCRIPTION
**Pull request by pellebjerkestrand**
This is an attempt to satisfy #17.

### CODE REVIEW

File: tools-of-quality.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## Issue Tracking
+
+A way to track issues must exist.
+
+The issues must be visible to, at least, the project members.
+
+The status of an issue must be visible on the issue itself, and reflect the actual status of the issue in the real world.
+
+This means that the overall state of the system is visible and knowable. E-mail, calls and other private messages are therefore not an acceptable form of issue tracking.
```
Comment (Reasonable-Solutions) [inline tools-of-quality.md]: <3

File: tools-of-quality.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+If an issue does not exist on the issue tracker, it does not (for the purposes of the system) exist.
+
+Work is derived from issues on the issue tracker by members suited to perform the work, not prescribed to specific members.
+
+We make no demands with regards to the exact type of issue tracker, but it must be compatible with the overall project workflow. JIRA, Trello, a whiteboard, Post-it Notes are all fine in different circumstances.
```
Comment (Reasonable-Solutions) [inline tools-of-quality.md]: > Jira, Trello, a whiteboard [...]

Perhaps "Github, Jira, Trello, a whit[...]", I think github deserves a mention

Comment (pellebjerkestrand) [reply tools-of-quality.md]: Feel free to add `GitHub` to the list.

File: tools-of-quality.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+For the sake of clarity we forgo the use of abbreviations unless these are widely understood (we will, for example, use `URL` instead of `Uniform Resource Locator`).
+
+We will write code, documentation, issues etc. in US English, as this is the language of the code we write, and makes it possible to include those not proficient in Norwegian as members of a team.
+
+If the customer demands that some of these things be done in Norwegian, an explicit choice must be made to satisfy that demand.
+
+Certain circumstances exist where mixing of languages is preferred, such as when a domain is only easily expressed in a given language.
+
+Regardless of language, we will strive to write and speak in a gramatically correct manner in order to avoid misunderstandings.
```
Comment (Reasonable-Solutions) [inline tools-of-quality.md:27]: Is this perhaps a bit superflous? 
I think we can grammar good and can learn to do other stuff good too?

Comment (pellebjerkestrand) [reply tools-of-quality.md:27]: You'd be surprised at how many people don't care at all. "Everyone knows what I mean" etc.

File: tools-of-quality.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+The issues must be visible to, at least, the project members.
+
+The status of an issue must be visible on the issue itself, and reflect the actual status of the issue in the real world.
+
+This means that the overall state of the system is visible and knowable. E-mail, calls and other private messages are therefore not an acceptable form of issue tracking.
+
+If an issue does not exist on the issue tracker, it does not (for the purposes of the project or system) exist.
+
+Work is derived from issues on the issue tracker by members suited to perform the work, not prescribed to specific members.
```
Comment (Reasonable-Solutions) [inline tools-of-quality.md]: "We strongly prefer pull based work" or something to that effect?

File: tools-of-quality.md (hunk 3 above)
Comment (pellebjerkestrand) [reply tools-of-quality.md:27]: See, for example, the whole "skjema" -> "schema" issue (found in several projects even though it doesn't make sense at all) 😱.

File: tools-of-quality.md (hunk 4 above)
Comment (pellebjerkestrand) [reply tools-of-quality.md]: Made a change to this.

File: tools-of-quality.md (hunk 3 above)
Comment (Reasonable-Solutions) [reply tools-of-quality.md:27]: :D

**Review by Reasonable-Solutions — APPROVED** [review]
This is starting to look pretty good! 
How merging and eh, cutting a release?

**Comment by pellebjerkestrand** [comment]
It would be cool if more members approved of this as well 😅

**Comment by Reasonable-Solutions** [comment]
@pellebjerkestrand a Monday Morning Meeting Mob Review, post-hoc , is perhaps in order (I'll be in sweden this upcomming monday though)

**Pull request merged**